tests/acceptance/ExportSubscribersCest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+use MailPoet\Test\DataFactories\Segment;
+
+require_once __DIR__ . '/../DataFactories/Segment.php';
+require_once __DIR__ . '/../_data/MailPoetExportList.csv';
+
+class ExportSubscribersCest {
+  function __construct() {
+    $this->search_field_element = 'input.select2-search__field';
+  }
+  function exportSubscribers(\AcceptanceTester $I) {
+    $segment_name = 'Hobbyists';
+    $segment = new Segment();
+    $segment->withName($segment_name)->create();
+    $I->wantTo('Export a list of subscribers');
+    $I->login();
+    $I->amOnMailPoetPage('Subscribers');
+    //import unique user list to export, so we have exact data regardless of other tests
+    $I->click(['xpath'=>'//*[@id="subscribers_container"]/div/h1/a[2]']);
+    $I->waitForText('Back to Subscribers', 10);
+    //select upload file as import method, import CSV
+    $I->click(['css'=>'#select_method > label:nth-of-type(2)']);
+    $I->attachFile(['css'=>'#file_local'], 'MailPoetExportList.csv');
+    $I->click(['xpath'=>'//*[@id="method_file"]/div/table/tbody/tr[2]/th/a']);
+    $I->waitForText('Jokūbas', 25);
+    //click is to trigger dropdown to display selections, least restrictive working method
+    $I->click($this->search_field_element);
+    $I->click(['xpath'=>'//*[@id="select2-mailpoet_segments_select-results"]/li[1]']);
+    //click next step
+    $I->click(['xpath'=>'//*[@id="step2_process"]']);
```

Comment:
For the basic selection of elements only based on ID, it may be easier to read if CSS selectors were used. E.g. compare

```
$I->click(['xpath'=>'//*[@id="step2_process"]']);
```
with
```
$I->click('#step2_process');
```

For more complicated cases - sure, using more specific or powerful selectors certainly makes sense.